test/e2e/storage/volume_provisioning.go

```diff
@@ -802,6 +798,15 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 				parameters:   map[string]string{"resturl": serverUrl},
 				attach:       false,
 			}
+
+			// GCE/GKE
+			if getDefaultPluginName() == "kubernetes.io/gce-pd" {
+				// using GB not GiB as e2e test unit since gce-pd returns GB,
+				// or expectedSize may be greater than claimSize.
+				test.claimSize = "2Gi"
```

Comment:
You could pass in "2G" for claim and expected sizes

Reply:
If the gluster provisioner does a roundup inherently, then should the test check for expected > claim, instead of equal?

Reply:
yes. comment above says use GB but units for claim are Gi. Seem confusing.

Reply:
thats a good thought, however as I am using common `testdynamicprovisioning()` func I am some how trying  to avoid extra conditions which I introduced. It looks like new size applied only for GCE actually helps in this direction. All the tests are passing including AWS with default size mentioned for these values. This looks bit more cleaner and avoid the `extra` condition in common func.

Reply:
Ok.. I am on it, will send the PR with updated comment.

Reply:
@msau42 done. added some more information.